mqtt/mqtt3/README.md

```diff
@@ -20,6 +20,31 @@ The crate is not published to crates.io yet. Please generate docs locally with `
 
 See the `examples/` directory for examples of a publisher and subscriber, and for how to set a will.
 
+# Fuzz testing
+
+The crate is coming with fuzz tests. 
+
+## Prerequisites
+
+```bash
+sudo apt install tmux binutils
+cargo install afl
+```
+
+## Run fuzzer
+
+```bash
+echo 'core' | sudo tee /proc/sys/kernel/core_pattern
+
+which gold
+
+/build/linux/mqtt-fuzz.sh ../mqtt3-fuzz/
```

Comment:
Add some explanation above each command. Something like:

```sh
# mqtt-fuzz.sh spawns a tmux session with 6 panes. The first pane is the master afl instance, and the remaining five are slave instances.
# Press any key to start the master instance, wait for it to start running, then press any key in the five slave instances to start them too.
```

and

```sh
# mqtt-fuzz-rerun.sh uses the output of a previous run of mqtt-fuzz.sh or mqtt-fuzz-rerun.sh as the starting corpus.
# Use this if you interrupted a previous run of mqtt-fuzz.sh or mqtt-fuzz-rerun.sh and want to resume from where it left off.
```

Reply:
Nice additions. Thanks!